PocketKit/Sources/PocketKit/MyList/ArchivedItemsList/ArchivedItemsListViewModel.swift

```diff
@@ -132,9 +132,8 @@ extension ArchivedItemsListViewModel {
     func overflowActions(for itemID: String) -> [ItemAction]? {
         return bareItem(with: itemID).flatMap { archivedItem in
             return [
-                .delete { [weak self] _ in
-                    self?.confirmDelete(item: archivedItem)
-                }
+                .delete { [weak self] _ in self?.confirmDelete(item: archivedItem) },
+                .reAdd { [weak self] _ in self?.reAdd(item: archivedItem) }
```

Comment:
Minor detail, but I'm wondering if the order of these should be flipped.

Reply:
Good call, flipped 'em